sql/hook/util_test.go

```diff
@@ -0,0 +1,53 @@
+package hook_test
+
+import (
+	"testing"
+
+	"github.com/ankorstore/yokai/sql/hook"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContains(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name string
+		list []string
+		item string
+		want bool
+	}{
+		{
```

Comment:
My general suggestion is always to cover corner cases such as a nil slice, an empty slice, or a slice with one item. :)

Reply:
Will push this in next commit 👍